py-polars/src/lazyframe/visitor/nodes.rs

```diff
@@ -344,7 +344,8 @@ pub(crate) fn into_py(py: Python<'_>, plan: &IR) -> PyResult<PyObject> {
                 #[cfg(feature = "ipc")]
                 FileScan::Ipc { .. } => return Err(PyNotImplementedError::new_err("ipc scan")),
                 #[cfg(feature = "json")]
-                FileScan::NDJson { options } => {
+                FileScan::NDJson { options, .. } => {
+                    // TODO: Also pass cloud_options
```

Comment:
I think this is for the GPU engine, this probably needs updates on that side as well so I've left a todo